app/controllers/async_info_controller.rb

```diff
@@ -71,6 +71,7 @@ def user_data
         moderator_for_tags: @user.moderator_for_tags,
         config_body_class: @user.config_body_class,
         pro: @user.pro?,
+        feed_style: SiteConfig.feed_style,
```

Comment:
Can we find a better place to serve this from? This hash is all comprised of `user_data` and this is a Site configuration. It feels like this should be its own standalone endpoint if we want a way to serve SiteConfigs to our views.

Reply:
The idea here is that this value will be overrideable by users in their `/settings/ux` so while that's not available yet, this will essentially be user information like anything else and this is just the default.

Reply:
_sigh_ fine, this still makes me cringe 😉